README.md

```diff
@@ -20,6 +20,7 @@ Dataset name | Organs | Staining | Link | Size | Data | Task | WSI/Patch | Other
 ACDC-LungHP [1a], [1b] | Lung | H&E | [data](https://acdc-lunghp.grand-challenge.org/), [paper](https://ieeexplore.ieee.org/document/9265237) | Train: 150, Test: 50 | images + xml | seg + classi | wsi | | 2019
 ACROBAT 2022 [66] | Breast | Multiple (IHC, H&E) | [data](https://acrobat.grand-challenge.org/), [paper](https://openreview.net/pdf?id=5TTocO2VI3r) | Train: 750 train; Valid: 100; Test: 300 | images (1 H&E match to 1-4 IHC) + landmarks | registration | wsi | 40x - Hamamatsu | 2022
 ADP [2] | multiple | multiple (most H&E) | [data](https://www.dsp.utoronto.ca/projects/ADP/), [github](https://github.com/mahdihosseini/ADP), [paper](https://openaccess.thecvf.com/content_CVPR_2019/papers/Hosseini_Atlas_of_Digital_Pathology_A_Generalized_Hierarchical_Histological_Tissue_Type-Annotated_CVPR_2019_paper.pdf) | Train: 14.134, Valid: 1767, Test: 1767 (100 wsi) | images + 57 hierarchical HTTs (histological tissue type) | multi-label (3) classification (hierarchy) | patch (1088x1088) | 40x - Huron TissueScope LE1.2 WSI | 2019
+AGGC | prostate | H&E | [data](https://aggc22.grand-challenge.org/AGGC22/) | Subset 1: train 105, test 45; Subset2: train 37 ,test 16; Subset3: train 144, test 67 | images scaned by different scanner | seg + gleason grading | wsi | Subset1 and Subset2: Akoya Biosciences Scanner, Subset3: each specimen is scanned by multiple scanners | 
```

Comment:
I would modify as followed:
AGGC | prostate | H&E | [data](https://aggc22.grand-challenge.org/AGGC22/), [paper](https://papers.ssrn.com/sol3/papers.cfm?abstract_id=4172090) | Subset 1: train 105, test 45; Subset2: train 37 ,test 16; Subset3: train 144, test 67 | images + binary masks | seg + gleason grading | wsi | 20x - Subset1 and Subset2: Akoya Biosciences Scanner, Subset3: each specimen is scanned by multiple scanners | 2022